src/blueprints/Do.Blueprints.Service.Application/Domain/Model/TypeModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Do.Domain.Model;
+
+public record TypeModel(Type Type, string Name,
```

Comment:
models can build themselves, lets create constructor overload `TypeModel(Type type)` to call this one